src/validation.cpp

```diff
@@ -725,8 +725,27 @@ class MemPoolAccept
 
 private:
     CTxMemPool& m_pool;
+
+    /** Holds a cached view of available coins from the UTXO set, mempool, and artificial temporary coins (to enable package validation).
+     * The view doesn't track whether a coin previously existed but has now been spent. We detect conflicts in other ways:
```

Comment:
> The view doesn't track whether a coin previously existed but has now been spent

been spent in mempool/package contexts, I presume

Reply:
s/has now been spent/has now been spent in the mempool/ ?

Reply:
Will change if I need to retouch